doc/contributing/security-release-process.md

```diff
@@ -223,6 +225,53 @@ out a better way, forward the email you receive to
   [Security release stewards](https://github.com/nodejs/node/blob/HEAD/doc/contributing/security-release-process.md#security-release-stewards).
   If necessary add the next rotation of the steward rotation.
 
+## Adding a security revert option
+
+Breaking changes are allowed in existing LTS lines in order to fix
+important security vulnerabilities. When breaking changes are made
+it is important to provide a command line option that restores
+the original behaviour.
+
+Node.js includes the command line option `--security-revert` and
+has the boilerplate to make additions easy.
```

Comment:
This starts by talking about LTS lines, but then switches context to `Node.js includes`. Upon first read, I assumed that this flag would only be included in LTS versions, missing the context switch. I'd recommend adding more, direct language about where and when this feature is available.